examples/thoughts/ext/ThoughtsPage.js

```diff
@@ -17,14 +18,17 @@ const ThoughtsPage = (props) => {
   const { data: thoughts } = useQuery(getThoughts, { tagName: tag })
 
   return (
-    <Layout
-      user={props.user}
-      activeTag={tag || '_all'}
-    >
-      <div className="center-container">
-        <ThoughtsList thoughts={thoughts} />
-      </div>
-    </Layout>
+    <>
+      <WaspSourceHeader name="Thoughts" />
+      <Layout
```

Comment:
We do have this Layout component in Thoughts app -> could we put WaspSourceHeader there, and nowhere else? We would be missing it on login and singup form, but it would be in the rest of the app, which is already pretty good, and we avoid duplication!